SwiftRustIntegrationTestRunner/integration-test-create-swift-package/src/main.rs

```diff
@@ -15,5 +15,6 @@ fn main() {
         )]),
         out_dir: PathBuf::from("swift-package-rust-library-fixture/MySwiftPackage"),
         package_name: "MySwiftPackage".to_string(),
+        resources: vec![],
```

Comment:
Can we add some resources?

- A file `resource-test.txt`
- A directory with 2 files `directory-with-files/{a,b}.txt
- A directory that contains a directory that contains a file `nested/directory/test.txt`

Then right after this `create_package` call we can call a `fn assert_resources_bundled`

Which checks the "swift-package-rust-library-fixture/MySwiftPackage" Sources dir to make sure that the resources are in there.

---

Also, if there's a way to check that the resources are accessible on the Swift side, that would be great. Could check in here https://github.com/chinedufn/swift-bridge/blob/a207c24d328b938f01642befe9c2be03ae3a7c5b/SwiftRustIntegrationTestRunner/swift-package-test-package/Tests/swift-package-test-packageTests/swift_package_test_packageTests.swift#L5-L17